fw/http.c

```diff
@@ -5824,8 +5812,10 @@ tfw_http_req_process(TfwConn *conn, TfwStream *stream, struct sk_buff *skb)
 		}
 	}
 
-	if ((r = tfw_http_req_process_authority_host(req)))
-		return r;
+	if (!__check_authority_correctness(req)) {
```

Comment:
I must pay your attention at this place. `__check_authority_correctness` will be called after parsing whole request, including body. For example: If we have request with invalid host/authority it will be dropped only after full parsing, however enough to parse only headers.